crates/pixi_build_discovery/src/discovery.rs

```diff
@@ -98,11 +98,11 @@ pub enum DiscoveryError {
     #[diagnostic(help("This is often caused by an internal error. Please report this issue."))]
     SpecConversionError(pixi_spec::SpecConversionError),
 
-    #[error("the source directory does not contain a supported manifest")]
+    #[error("the source directory '{0}', does not contain a supported manifest")]
```

Comment:
Added this so that it can help the user figure out what is going wrong:
Before:
```
❯ pixi global install --git https://github.com/wolfv/pixi-build-examples test
Error:   × Couldn't install test
  ├─▶ failed to extract metadata for package 'test'
  ╰─▶ the source directory does not contain a supported manifest
```
After:
```
➜ pixi global install --git https://github.com/wolfv/pixi-build-examples boost-check                   
Error:   × Couldn't install boost-check
  ├─▶ failed to extract metadata for package 'boost-check'
  ╰─▶ the source directory '/Users/rubenarts/Library/Caches/rattler/cache/git-v0/checkouts/8b82163feaa7a7cc/1cb2663', does not contain a supported manifest
```
While the checkout might be awkward, it allowed me to inspect the checkout and validate the name/structure.

Reply:
Looks good!